Cargo.toml

```diff
@@ -45,6 +45,7 @@ source-update-bindings = ["build-bindgen"]
 cty = {version = "0.2", optional = true}
 
 [build-dependencies]
+version_check = "0.1"
```

Comment:
I didn't know about this crate. It seems pretty cool - I'll have to look into it for my own projects.